app/helpers/moments_form_helper.rb

```diff
@@ -65,30 +65,36 @@ def moment_strategy
   def moment_comment
     moment_input_props('comment', 'switch', 'comment.allow_comments')
       .merge(value: true,
-             uncheckedValue: false,
-             checked: @moment.comment,
-             info: t('comment.hint'),
-             dark: true)
+             uncheckedValue: false, checked: @moment.comment,
+             info: t('comment.hint'), dark: true)
   end
 
   def moment_publishing
-    { id: 'moment_publishing',
-      type: 'switch',
+    { id: 'moment_publishing', type: 'switch',
       label: t('moments.form.draft_question'),
-      dark: true,
-      name: 'publishing',
-      value: '0',
-      uncheckedValue: '1',
+      dark: true, name: 'publishing',
+      value: '0', uncheckedValue: '1',
       checked: !@moment.published? }
   end
 
+  def moment_bookmarked
+    moment_input_props('bookmarked', 'switch', 'moments.bookmark')
+      .merge(
+        value: true,
+        uncheckedValue: false,
+        checked: @moment.bookmarked,
+        dark: true
+      )
+  end
+
   def moment_form_inputs
     [
       moment_name, moment_why, moment_fix, moment_category, moment_mood,
       moment_strategy, get_viewers_input(
         @viewers, 'moment', 'moments', @moment
       ),
-      moment_comment, moment_publishing
+      moment_comment, moment_publishing,
+      Rails.env.development? ? moment_bookmarked : ''
```

Comment:
Instead of returning an empty string if the env if production, I think it should be nil.

Reply:
Hi @julianguyen, some tests seem to fail when I make this change. I had noticed this previously and hence replaced it with an empty string. Is there another way to solve this?